invenio_app_ils/ui/src/invenio_app_ils/common/api/loans/loan.js

```diff
@@ -195,6 +195,12 @@ const list = query => {
   });
 };
 
+const postEmail = async payload => {
+  const url = generatePath(ApiURLS.loans.email, { loanPid: payload.loanPid });
+  const response = await http.post(url, payload);
```

Comment:
try/catch with nice error message?

Reply:
No other function has try/catch in the api, I left a comment above about doing it in the mail confirmation modal component itself.